specs/lightclient/beacon-chain.md

```diff
@@ -155,6 +157,8 @@ class BeaconState(Container):
     # Light client sync committees
     current_sync_committee: SyncCommittee
     next_sync_committee: SyncCommittee
+    # Denominator to real-time-updated balances (NOT effective balances!)
+    balance_denominator: uint64
```

Comment:
((porting previous comments to here))
https://github.com/ethereum/eth2.0-specs/pull/2150/files#r550251888
@dankrad: This means balances is not denoted in `GWei` anymore, should be changed to `uint64`